src/sentry/api/urls.py

```diff
@@ -1688,6 +1691,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         OrganizationProjectsEndpoint.as_view(),
         name="sentry-api-0-organization-projects",
     ),
+    re_path(
+        r"^(?P<organization_slug>[^\/]+)/lightweight-projects/$",
```

Comment:
```suggestion
        r"^(?P<organization_slug>[^\/]+)/projects/options/$",
```

Since this list of projects is more for building option lists we could use that naming or another similar term instead of weight.

Reply:
This is a better name!
It's slightly a little bit about the intended use, another use will be to almost replace AvatarProject in the few places it's used.

Reply:
I am confused about this naming change, as we don't actually return the project options from this endpoint

Reply:
changed it to minimal projects lmk if that is okay.